Docker/alma9/push.sh

```diff
@@ -1,5 +0,0 @@
-docker tag alma9 gitlab-registry.cern.ch/key4hep/k4-deploy/alma9
```

Comment:
Why is this deleted? These images are also pushed to the CERN gitlab and are the ones used to build the Key4hep stack. In addition now building the images with cvmfs has forcefully to go through Github. Can it not be like before when the local one is reused?